From 50b7344bc2cc941a43b3adfaebdc96dd1935e747 Mon Sep 17 00:00:00 2001
From: Jonathan Ross Rogers <jrogers@socialserve.com>
Date: Mon, 9 May 2011 18:16:29 -0400
Subject: [PATCH] Cause qgit to use "git config" everywhere.

Whenever starting up qgit, it pops up a dialog showing that git
complained about the obsolete command "repo-config". This patch
replaces remaining calls to "git repo-config" with "git config".

Signed-off-by: Jonathan Ross Rogers <jrogers@socialserve.com>
---
 src/git.cpp |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/git.cpp b/src/git.cpp
index 50e6dd3..8e9365c 100644
--- a/src/git.cpp
+++ b/src/git.cpp
@@ -427,7 +427,7 @@ void Git::setTextCodec(QTextCodec* tc) {
 	if (name == "Big5-HKSCS")
 		name = "Big5";
 
-	run("git repo-config i18n.commitencoding " + name);
+	run("git config i18n.commitencoding " + name);
 }
 
 QTextCodec* Git::getTextCodec(bool* isGitArchive) {
@@ -437,7 +437,7 @@ QTextCodec* Git::getTextCodec(bool* isGitArchive) {
 		return NULL;
 
 	QString runOutput;
-	if (!run("git repo-config --get i18n.commitencoding", &runOutput))
+	if (!run("git config --get i18n.commitencoding", &runOutput))
 		return NULL;
 
 	if (runOutput.isEmpty()) // git docs says default is utf-8
-- 
1.7.5

